terraform/README.md

```diff
@@ -0,0 +1,37 @@
+# Deploy Online Boutique sample application on a GKE cluster
+
+This page walks you through the steps required to deploy the [Online Boutique](https://github.com/GoogleCloudPlatform/microservices-demo) sample application on a GKE ([Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine)) cluster using Terraform.
+
+## Pre-requisites
+
+Setting up the sample requires that you have a [Google Cloud Platform (GCP) project](https://cloud.google.com/resource-manager/docs/creating-managing-projects#console), connected to your billing account.
+
+## Deploying the sample application
+
+Once you have ensured that all the pre-requisites are met, follow the steps below to create a GKE cluster and deploy the Online Boutique sample application:
+
+1. Clone the repo:
+`git clone https://github.com/GoogleCloudPlatform/microservices-demo.git`
+1. Move into the Terraform directory that has the installation scripts:
+`cd microservices-demo/terraform`
```

Comment:
for commands that you want people to be able copy pasta, I would recommend changing this (and the other copy pasta-able cmds) to :
```
cd microservices-demo/terraform
```

Reply:
sorry i just saw that it formatted, what i meant is having the 3 backticks ``` surrounding the text